src/app/nft-post-page/nft-post/nft-post.component.html

```diff
@@ -215,6 +216,7 @@
               <i class="fas fa-check-circle fa-md align-middle"></i>
             </span>
                 </div>
+                <div class="fc-red" *ngIf="owner.IsPending"><i>Pending</i></div>
```

Comment:
Do we want this to be red? Maybe just gray or something?

Reply:
yeah gray is less intimidating
![image](https://user-images.githubusercontent.com/81658138/148254108-24c90c3a-a1d7-4eb4-9737-afa799209ac1.png)